mindsdb/integrations/handlers/bard_handler/README.md

```diff
@@ -0,0 +1,65 @@
+# Bard Handler
+The Bard ML handler integrates the Bard library from Google's Generative AI with MindsDB. You can use it to generate text completions with the Gemini Pro model for your existing text data.
+
+## Google Generative AI (Bard)
+Google Generative AI is a library that provides access to powerful language models for text generation. The Bard library, used in this handler, offers various models for generating text. More information about the Google Generative AI library can be found [here](https://github.com/GoogleCloudPlatform/generative-ai).
```

Comment:
I don't see a Bard library in the link. Furthermore, if the underlying integration is to Gemini models I would request that you please rename this integration to "google_gemini_handler" (don't worry about the original issue, we can modify it afterwards).

Reply:
Yes sure will make the change